src/backend/src/modules/pia-intake/metadata/pia-status.metadata.ts

```diff
@@ -283,8 +283,133 @@ export const piaStatusMetadata: Partial<
           },
         ],
       },
+      [PiaIntakeStatusEnum.PENDING_COMPLETION]: {
+        allow: true,
+        conditions: [
+          {
+            piaType: [PiaTypesEnum.DELEGATE_REVIEW],
+            fieldValidations: {
+              'review.mpo.reviewNote': {
+                required: true,
+              },
+              'review.programArea.reviews': {
+                required: true,
+                size: SizeEnum.ALL,
+              },
+            },
+          },
+          {
+            piaType: [PiaTypesEnum.STANDARD],
+            fieldValidations: {
+              'review.mpo.reviewNote': {
+                required: true,
+              },
+              'review.cpo': {
+                required: true,
+              },
+              'review.programArea.reviews': {
+                required: true,
+                size: SizeEnum.ALL,
+              },
+            },
+          },
+        ],
+      },
+    },
+  },
+  [PiaIntakeStatusEnum.PENDING_COMPLETION]: {
+    updates: {
+      allow: false,
+    },
+    transition: {
+      [PiaIntakeStatusEnum.INCOMPLETE]: {
+        allow: true,
+        conditions: [
+          {
+            accessType: [UserTypesEnum.CPO],
+          },
+        ],
+        actions: [
+          {
+            type: 'update',
+            key: 'review',
+            value: null,
+          },
+        ],
+      },
+      [PiaIntakeStatusEnum.EDIT_IN_PROGRESS]: {
+        allow: true,
+        conditions: [
+          {
+            accessType: [UserTypesEnum.CPO],
+          },
+        ],
+        actions: [
+          {
+            type: 'update',
+            key: 'review',
+            value: null,
+          },
+        ],
+      },
+      [PiaIntakeStatusEnum.MPO_REVIEW]: {
+        allow: true,
+        conditions: [
+          {
+            accessType: [UserTypesEnum.CPO],
+          },
+        ],
+      },
+      [PiaIntakeStatusEnum.CPO_REVIEW]: {
+        allow: true,
+        conditions: [
+          {
+            accessType: [UserTypesEnum.CPO],
+          },
+        ],
+      },
+      [PiaIntakeStatusEnum.FINAL_REVIEW]: {
+        allow: true,
+        conditions: [
+          {
+            accessType: [UserTypesEnum.CPO],
+          },
+        ],
+      },
       [PiaIntakeStatusEnum.COMPLETE]: {
         allow: true,
+        conditions: [
+          {
+            accessType: [UserTypesEnum.CPO],
+          },
```

Comment:
I doubt this is true. Any CPO user should not be able to move from Pending Completion to COMPLETE without conditions. Suggested addition of User Types to below conditions. And recommend removing this one.